### ISSUE TITLE
NullReference

### ISSUE THREAD

**Issue by viktorx11** [cDima/Aerial#36]
![image](https://cloud.githubusercontent.com/assets/13213801/11531566/1146c852-98fe-11e5-8e6c-28a3ed332ced.png)

**Comment by cDima** [comment]
Yes, I noticed this error sometimes, but it seems to be wrapped inside of the Windows Media Player exe, would need help to identify when not to call SetNextVideo.

**Comment by viktorx11** [comment]
This one is gone with the new version v3.1, at least for me.
In v3 it came every time i start the preview of the screensaver

**Comment by cDima** [comment]
@viktorx11 please check latest version to see if this is also fixed.
